sh.loft.Devpod.metainfo.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>sh.loft.Devpod</id>
```

Comment:
why is this `sh.loft` when the website is `devpod.sh`?

Reply:
Loft is the organisation, devpod is the product. We own `loft.sh` as well as `devpod.sh`, I named it this in case we wanted to use `sh.loft.vcluster` (our other product)